### PR TITLE
split validators for different lambda functions

### PR DESCRIPTION
**Pull request by bmatern**

### CODE REVIEW

File: SchemaValidator/ValidationCommon.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+def updateValidationStatus(fileName=None, isValid=None, validationFeedback=None, token=None, url=None):
+    if(url is None):
+        url=getUrl()
+    if(fileName is None):
+        print('No filename was provided, I cannot set validation status.')
+        return False
+    try:
+        fullUrl = str(url) + '/api/uploads/setvalidation'
+        # TODO: Haml Files?
+        body = {'valid': isValid, 'validationFeedback': validationFeedback, 'fileName':fileName, 'type':'HML'}
```
Comment (cyt-mn) [inline SchemaValidator/ValidationCommon.py]: new object format is like 
{
'valid': true,
'validationFeedback': 'asdf',
'validator': 'MIRING',
'upload': {
  'fileName': 'asdf.hml'
  }
}

File: SchemaValidator/ValidationHandlers.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+        setValidationStatus(uploadFileName=xmlKey, isValid=(validationResults=='Valid'), validationFeedback=validationResults, url=url, token=token)
+
+        return str(validationResults)
+
+    except Exception as e:
+        print('Exception:\n' + str(e) + '\n' + str(exc_info()))
+        return str(e)
+
+def miring_validation_handler(event, context):
```
Comment (cyt-mn) [inline SchemaValidator/ValidationHandlers.py]: So we deploy basically the same code, but with different entry functions? Wouldn't it be better to split this into a miringHandler.py and a xmlSchemaHandler.py? Or does this cause trouble anywhere else?

**Review by cyt-mn — CHANGES_REQUESTED** [review]
@bmatern please check my two comments. Besides from that, I guess we're good.

File: SchemaValidator/ValidationCommon.py (hunk 1 above)
Comment (bmatern) [reply SchemaValidator/ValidationCommon.py]: Okay, making this change.

File: SchemaValidator/ValidationHandlers.py (hunk 2 above)
Comment (bmatern) [reply SchemaValidator/ValidationHandlers.py]: Yeah I think that's fine. I put the handlers in one file and the validators in each other file. Im changing to one file per validator, each has it's own handler. Deployment for each lambda will be two files (ex SchemaValidation.py and ValidationCommon.py) and also the configuration file.

**Pull request merged**